source/main.ts

```diff
@@ -133,9 +133,10 @@ for (const endpoint of args['--listen']) {
       await clipboard.write(local);
       message += `\n\n${chalk.grey('Copied local address to clipboard!')}`;
     } catch (error: unknown) {
-      logger.error(
-        `Cannot copy server address to clipboard: ${(error as Error).message}.`,
-      );
+      if (args['--debug'])
```

Comment:
Is there any way to tell this is being used in an environment where this isn't wanted?

Also, could we add a test please?